app/views/layouts/base.html.erb

```diff
@@ -41,7 +41,9 @@
     <%= govuk_footer(
       meta_items_title: 'Footer links',
       meta_items: {
+        'Accessibility': '/accessibility',
         'Cookies': '/cookies',
+        'Privacy': '/privacy',
```

Comment:
I think we should comment the links out. We should be careful about making the service "look complete" when actually some of it is placeholder, as someone who isn't familiar might say "okay it's done" and we might ship it before it's ready.

Reply:
I figured the HTTP basic authentication prevented us from thinking the service looked complete, but happy to comment them out if you think that's worth it.

Reply:
Maybe it's fine. Just keep it in mind come go-live that someone still has to do a pass for those. :)